packages/forms-web-app/src/routes/index.js

```diff
@@ -6,8 +6,10 @@ const appellantSubmissionRouter = require('./appellant-submission');
 const eligibilityRouter = require('./eligibility');
 const homeRouter = require('./home');
 const cookieRouter = require('./cookies');
+const guidancePagesRouter = require('./guidance-pages');
 
 router.use('/', homeRouter);
+router.use(guidancePagesRouter);
```

Comment:
Does this need an endpoint?

Reply:
Nope, endpoints from the router and it's at base route